Add a little positive note in the intro about Carbon (#270)
readthedocs/ethicalads.io: content/pages/vs-carbon-ads.md

This page talks a bit more about how we're different,
and why you might want to choose us instead of Carbon Ads.

> Carbon and their parent company BuySellAds have attractive, unobtrusive ads
> and like us also focus on developers and related niches.
> However, there are some differences between EthicalAds and Carbon
> especially around privacy, security, and community focus.
> Let's explore them.

Reviewer: ```suggestion
Carbon and their parent company BuySellAds have attractive, unobtrusive ads.
Like us, they also focus on developers and related niches.
We are happy to compete with company that also cares about the user experience around ads.
However, there are some differences between EthicalAds and Carbon
especially around privacy, security, and community focus.
Let's explore them.
```

Author: I made a minor typo edit (happy to compete with **a** company) on the above but it looks good.

[TOC]

## **Privacy**: We never set a cookie on your site, or load third-party scripts.

> When users go to your website, they are assuming they will only be sending data to you. Some users consider it a breach of trust if you were to share that data with other third parties, and it might be a breach of a privacy law to do this without user consent.

Author: I don't love this paragraph overall, to be honest since we are also a third-party to the user. However, I think there is a way to make that distinction between a resource loaded by the publisher and a resource loaded by a third-party.

Reviewer: Heh yea, valid point.

> When users go to your website, they are assuming they will only be sending data to you. Some users consider it a breach of trust if you were to share that data with other third parties, and it might be a breach of a privacy law to do this without user consent.

Reviewer: ```suggestion
When users go to your website, they are expecting to send data to you and the provider you choose. When using a provider like Carbon Ads, they will also send your data to third-parties that you didn't even know about. EthicalAds will never load javascript from other providers on your pages. 
```

Author: Maybe, "When running a web site, if you integrate with an ad network, you expect to send data to that network."?

Reviewer: Sure, that works 👍

> Some ad networks will set cookies on your browser session, as well as load scripts or resources hosted by third parties into a publishers web page. You can see a screenshot of Carbon Ads loading a Google tracking pixel here:

Reviewer: Make it more explicit? I think `Some ad networks` was left over from focusing on multiple providers in this page. 

```suggestion
Carbon Ads reserves the right to set cookies on your browser session, as well as load scripts or resources hosted by third parties into a publishers web page. You can see a screenshot of Carbon Ads loading a Google tracking pixel here:
```
